core/templates/dev/head/pages/exploration-editor-page/statistics-tab/issues/multiple-incorrect-submissions-issue.directive.ts

```diff
@@ -0,0 +1,103 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the Multiple Incorrect Submission Issue.
+ */
+
+require('services/exploration-html-formatter.service.ts');
+require('services/html-escaper.service.ts');
+require('domain/utilities/url-interpolation.service.ts');
+require(
+  'pages/exploration-editor-page/statistics-tab/services/' +
+  'learner-action-render.service.ts');
+
+angular.module('oppia').directive('multipleIncorrectSubmissionsIssue', [
+  'ExplorationHtmlFormatterService', 'ExplorationStatesService',
+  'HtmlEscaperService', 'LearnerActionRenderService', 'UrlInterpolationService',
+  'ACTION_TYPE_ANSWER_SUBMIT',
+  function(
+      ExplorationHtmlFormatterService, ExplorationStatesService,
+      HtmlEscaperService, LearnerActionRenderService, UrlInterpolationService,
+      ACTION_TYPE_ANSWER_SUBMIT) {
+    return {
+      restrict: 'E',
+      scope: {},
+      bindToController: {},
+      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
+        '/pages/exploration-editor-page/statistics-tab/issues/' +
+        'multiple-incorrect-submissions-issue.directive.html'),
+      controllerAs: '$ctrl',
+      controller: ['$attrs', function($attrs) {
+        var ctrl = this;
+        ctrl.finalBlock = HtmlEscaperService.escapedJsonToObj(
+          $attrs.finalBlock);
+        ctrl.index = this.finalBlock.length - 1;
+        ctrl.actionStartIndex = $attrs.actionStartIndex;
+        ctrl.stateName =
+          this.finalBlock[this.index].actionCustomizationArgs.state_name.value;
+
+        var _nonSubmitActions = [];
+        for (
+          var i = 0;
+          this.finalBlock[i].actionType !== ACTION_TYPE_ANSWER_SUBMIT; i++) {
+          _nonSubmitActions.push(this.finalBlock[i]);
+        }
+        ctrl.nonSubmitActions = _nonSubmitActions;
+        ctrl.tableIndex = +this.actionStartIndex + i;
+
+        ctrl.getShortAnswerHtml = function(action) {
+          var _answer = action.actionCustomizationArgs.submitted_answer.value;
+          var interaction = ExplorationStatesService.getState(
+            this.stateName).interaction;
+          return ExplorationHtmlFormatterService.getShortAnswerHtml(
+            _answer, interaction.id, interaction.customizationArgs);
+        };
+
+        ctrl.getFeedback = function(action) {
+          var _feedback = action.actionCustomizationArgs.feedback.value._html;
+          _feedback = _feedback.replace(
+            '{{answer}}', this.getShortAnswerHtml(action));
+          return _feedback;
+        };
+
+        ctrl.isActionSubmit = function(action) {
+          return action.actionType === ACTION_TYPE_ANSWER_SUBMIT;
+        };
+
+        /**
+         * Returns the index of the learner action.
+         * @param {LearnerAction} learnerAction.
+         * @returns {int}
+         */
+        ctrl.getLearnerActionIndex = function(learnerAction) {
```

Comment:
Ditto -- keep track of action indexes and translate to actions, not the other way around.

Reply:
Done.